doc/man3/RAND_DRBG_set_callbacks.pod

```diff
@@ -0,0 +1,106 @@
+=pod
+
+=head1 NAME
+
+RAND_DRBG_set_callbacks,
+RAND_DRBG_get_entropy_fn,
+RAND_DRBG_cleanup_entropy_fn,
+RAND_DRBG_get_nonce_fn,
+RAND_DRBG_cleanup_nonce_fn
+- set callbacks for reseeding
+
+=head1 SYNOPSIS
+
+ #include <openssl/rand_drbg.h>
+
+
+ int RAND_DRBG_set_callbacks(RAND_DRBG *drbg,
+                             RAND_DRBG_get_entropy_fn get_entropy,
+                             RAND_DRBG_cleanup_entropy_fn cleanup_entropy,
+                             RAND_DRBG_get_nonce_fn get_nonce,
+                             RAND_DRBG_cleanup_nonce_fn cleanup_nonce);
+
+
+=head2 Callback Functions
+
+ typedef size_t (*RAND_DRBG_get_entropy_fn)(
+                      RAND_DRBG *drbg,
+                      unsigned char **pout,
+                      int entropy, size_t min_len,
+                      size_t max_len);
+
+ typedef void (*RAND_DRBG_cleanup_entropy_fn)(
+                      RAND_DRBG *drbg,
+                      unsigned char *out, size_t outlen);
+
+ typedef size_t (*RAND_DRBG_get_nonce_fn)(
+                      RAND_DRBG *drbg,
+                      unsigned char **pout,
+                      int entropy,
+                      size_t min_len, size_t max_len);
+
+ typedef void (*RAND_DRBG_cleanup_nonce_fn)(
+                      RAND_DRBG *drbg,
+                      unsigned char *out, size_t outlen);
```

Comment:
`doc-nits` reacts on these four typedefs, because you need to add them to `util/private.num`.  `grep '^OSSL_STORE.*_fn util/private.num` shows you other similar examples.

Reply:
Or more properly if they are public typedefs then they probably have to be formatted specially.  See the regexp in util/find-doc-nits.

Reply:
Thanks, I will check it this evening.